backend/Match/serializers.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework import serializers
+from .models import Match
+
+class MatchSerializer(serializers.ModelSerializer):
+	state_display = serializers.CharField(source='get_state_display', read_only=True)
+
+	
+	# tournament_id = serializers.PrimaryKeyRelatedField(queryset=Tournament.objects.all())
+	# tournament_phase_id = serializers.PrimaryKeyRelatedField(queryset=TournamentPhase.objects.all())
+
+	class Meta:
+		model = Match
+		fields = ['id', 'state', 'state_display', 'start_time', 'end_time', 'created_at', 'updated_at']
```

Comment:
insert_ts, updated_ts to follow the convention with previously used columns that present the same thing!

Reply:
ok, but wouldn't you rather change the current insert_ts-s and updated_ts-s into created_at and updated_at?

Reply:
it's a bit weird that all the other fields don't have capitalised letters, but then we have something like insertTS for the User and Friends models XD

Reply:
_ts aka timestamp, which tell us what kind of data we hold in the column, same with start_ts and end_ts or smt would be good, unless Django supports getting just the time

Reply:
In anycase I would indeed change either the existing data or these columns so that all columns that hold timestamps / datetime values for when the row was crreated / updated, will look the same. Its really for the other coders to know what this column does and we should really try to hold somekind of conventions. Think about it, if we have a column called 'name' in one table and then on another table something like 'called_by' or smt, would be confusing at least for me to understand that these both columns mean the name of the specific row.

Reply:
ok, I'll make an issue and branch where I changed all the models' time fields so that they all have the FOO_ts format.